docs/cis_self_assessment.md

```diff
@@ -355,225 +351,106 @@ root:root
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chown -R root:root /etc/kubernetes/ssl
-```
-
 **Audit:**
-
-```
-stat -c %U:%G /etc/kubernetes/ssl
+```bash
+stat -c %U:%G /var/lib/rancher/rke2/server/tls
+root:root
 ```
 
-**Expected result**:
-
-```
-'root:root' is present
+**Remediation:**
+```bash
+chown -R root:root /var/lib/rancher/rke2/server/tls
 ```
 
 
 #### 1.1.20
 Ensure that the Kubernetes PKI certificate file permissions are set to `644` or more restrictive (Scored)
 <details>
 <summary>Rationale</summary>
-Kubernetes makes use of a number of certificate files as part of the operation of its components. The permissions on these files should be set to 644 or more restrictive to protect their integrity.
+RKE2 makes use of a number of certificate files as part of the operation of its components. The permissions on these files should be set to 644 or more restrictive to protect their integrity. This is a manual check and each file in the output will need to be verified.
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chmod -R 644 /etc/kubernetes/ssl
-```
-
-**Audit Script:** check_files_permissions.sh
-
-```
-#!/usr/bin/env bash
-
-# This script is used to ensure the file permissions are set to 644 or
-# more restrictive for all files in a given directory or a wildcard
-# selection of files
-#
-# inputs:
-#   $1 = /full/path/to/directory or /path/to/fileswithpattern
-#                                   ex: !(*key).pem
-#
-#   $2 (optional) = permission (ex: 600)
-#
-# outputs:
-#   true/false
-
-# Turn on "extended glob" for use of '!' in wildcard
-shopt -s extglob
-
-# Turn off history to avoid surprises when using '!'
-set -H
-
-USER_INPUT=$1
-
-if [[ "${USER_INPUT}" == "" ]]; then
-  echo "false"
-  exit
-fi
-
-
-if [[ -d ${USER_INPUT} ]]; then
-  PATTERN="${USER_INPUT}/*"
-else
-  PATTERN="${USER_INPUT}"
-fi
-
-PERMISSION=""
-if [[ "$2" != "" ]]; then
-  PERMISSION=$2
-fi
-
-FILES_PERMISSIONS=$(stat -c %n\ %a ${PATTERN})
-
-while read -r fileInfo; do
-  p=$(echo ${fileInfo} | cut -d' ' -f2)
-
-  if [[ "${PERMISSION}" != "" ]]; then
-    if [[ "$p" != "${PERMISSION}" ]]; then
-      echo "false"
-      exit
-    fi
-  else
-    if [[ "$p" != "644" && "$p" != "640" && "$p" != "600" ]]; then
-      echo "false"
-      exit
-    fi
-  fi
-done <<< "${FILES_PERMISSIONS}"
-
-
-echo "true"
-exit
-```
-
-**Audit Execution:**
-
-```
-./check_files_permissions.sh '/etc/kubernetes/ssl/\*.pem'
+**Audit:** 
+```bash
+stat -c %n\ %a /var/lib/rancher/rke2/server/tls/*.crt
 ```
-<!-- TODO escaped asterisk -->
-**Expected result**:
 
-```
-'true' is present
+**Remediation:**
```

Comment:
Same comment as above regarding remediation. I want to highlight that its fixed by default. if people want the manual remediation, they can go to the upstream CIS guide.